framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -113,38 +124,35 @@ public ValueTransfer(CFAbstractAnalysis<CFValue, CFStore, CFTransfer> analysis)
         return ValueAnnotatedTypeFactory.getCharValues(intAnno);
     }
 
+    /** Returns a list of possible values, or null if no estimate is available. */
```

Comment:
This method returns null when the type of subNode is @BottomVal. This seems to go against convention.